packages/tools/locales/fr_FR.po

```diff
@@ -484,9 +486,9 @@ msgid ""
 "for to-dos, or `nt` for notes and to-dos (eg. `-tt` would display only the "
 "to-dos, while `-tnt` would display notes and to-dos."
 msgstr ""
-"Affiche uniquement les notes du ou des types spécifiés. Le type peut-être "
-"`n` pour les notes, `t` pour les tâches (par exemple, `-tt` affiche "
-"uniquement les tâches, tandis que `-tnt` affiche les notes et les tâches)."
+"Affiche uniquement les notes du ou des types spécifiés. Le type peut‑être "
+"`n` pour les notes, `t` pour les tâches (par exemple, `‑tt` affiche "
+"uniquement les tâches, tandis que `‑tnt` affiche les notes et les tâches)."
```

Comment:
Please don't change flags as it means if a user copy and paste them they won't work.

Reply:
I noticed that mistake, this is now corrected. I check the whole file; no other flags have been changed.